.changeset/wicked-melons-study.md

```diff
@@ -0,0 +1,6 @@
+---
+'@astrojs/starlight': minor
+---
+
+Expands the `<Badge>` component to be generally available
```

Comment:
I know we think of this as “expanding”, but I actually think we can describe this as “adding” a component — the previous component doesn’t really exist for users currently, they only enable it via config options.

```suggestion
Adds a new `<Badge>` component
```